runtime-library/create_xcframework.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+SCRIPT_DIR=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
+IREE_HOST_BUILD_DIR=$SCRIPT_DIR/../../iree-build/install/bin
```

Comment:
I suspect that we ultimately want to make this script "hermetic" so that it is a one-stop build of a framework. For that case, it probably makes sense to create local build directories and set it up exactly as needed vs relying on the developer having prepped something externally. I'm more interested in this working for now vs being perfect, so just a thought for the future.

Reply:
Makes total sense!